index.js

```diff
@@ -454,6 +454,37 @@ var handleGuardDutyFinding = function(event) {
   return _.merge(slackMessage, baseSlackMessage);
 };
 
+var handleEC2RetirementScheduled = function(event) {
+
+  var subject = event.Records[0].Sns.Subject;
+  var timestamp = new Date(event.Records[0].Sns.Timestamp).getTime() / 1000;
+  var message = JSON.parse(event.Records[0].Sns.Message);
+  var color = "warning";
+
+  var slackMessage = {
+    text: "*" + subject + "*",
+    attachments: [
+      {
+        "color": color,
+        "fields": [
+          { "title": "Account", "value": deriveAccountName(message.account), "short": true },
+          { "title": "Region", "value": message.region, "short": true },
+          { "title": "Event", "value": message.detail['eventTypeCode'], "short": true },
+          { "title": "Resource", "value": message.resources[0], "short": true },
+          { "title": "EventArn", "value": message.detail['eventArn'], "short": true },
+          {
+            "title": "Link to Event",
+            "value": " https://" + message.region + ".console.aws.amazon.com/ec2/v2/home?region=" + message.region + "#Events:",
```

Comment:
I noticed the test message didn't have the the `:` included at the end of the URL, maybe this extra space before ` https` confused it.

Reply:
when concatenating more than a few things, it might be a good chance to use string interpolation if this is running a new enough version of nodejs:
```
"value": `https://${message.region}.blah.com?region=${message.region}#zing`,
```

Reply:
OMG, luckily it still can work

Reply:
The string interpolation is also not work for `:` :(